generated/sql/microsoft.sql/2021-11-01-preview/types.md

```diff
@@ -1591,7 +1591,7 @@ For more information, see [Auditing to storage using Managed Identity authentica
 ### Properties
 * **autoRotationEnabled**: bool: Key auto rotation opt-in flag. Either true or false.
 * **serverKeyName**: string: The name of the server key.
-* **serverKeyType**: 'AzureKeyVault' | 'ServiceManaged' (Required, WriteOnly): The encryption protector type like 'ServiceManaged', 'AzureKeyVault'.
```

Comment:
There are a few `WriteOnly` flags removed from this file, but I haven't located any other such changes. I can't find anything in the commit logs of this repository or in those of `azure-rest-api-specs`, so I presume that this was caused by an update in Autorest or ModelerFour.